.github/workflows/msys2.yml

```diff
@@ -0,0 +1,72 @@
+name: msys2
+
+on:
+  push:
+    branches:
+      - "master"
+    paths-ignore:
+      - "**.md"
+  pull_request:
+    branches:
+      - "master"
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    defaults:
+      run:
+        shell: msys2 {0}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { sys: mingw64, env: x86_64 }
+          - { sys: mingw32, env: i686 }
+
+    steps:
+      - name: Install pandoc
+        shell: cmd
+        run: choco install pandoc
+
+      - name: Extract Version Number
```

Comment:
Also, we no longer need the "Extract Version Number" step, it was used in the "Release" step.

Reply:
It shouldn't hurt and it keeps the diff to Crispy smaller.

Reply:
But it does nothing. We can use this as an opportunity to update main.yml and possibly integrate MSYS2 into it. I made a separate msys2.yml file to minimize the diff too 😄

Reply:
I merged this to get a windows build in CI to avoid any further unexpected breakage. If you want to tweak it more feel free to send a new PR.